include/benchmark/reporter.h

```diff
@@ -84,15 +86,21 @@ class BenchmarkReporter {
     static void ComputeStats(std::vector<Run> const& reports, Run* mean, Run* stddev);
 };
 
+typedef std::pair<const char*,double> TimeUnitMultiplier;
+
 // Simple reporter that outputs benchmark data to the console. This is the
 // default reporter used by RunSpecifiedBenchmarks().
 class ConsoleReporter : public BenchmarkReporter {
  public:
   virtual bool ReportContext(const Context& context);
   virtual void ReportRuns(const std::vector<Run>& reports);
-protected:
+
+ protected:
   virtual void PrintRunData(const Run& report);
 
+ private:
+  TimeUnitMultiplier getTimeUnitAndMultiplier(TimeUnit unit);
```

Comment:
consistent naming style please: GetTimeUnitAndMultiplier.

Further, it doesn't need to be a member of any class as it doesn't use state from the class. It doesn't even need to be in `include/benchmark` as it'll only be called from implementations of reporters.